spec/rakelib/support/cookies_spec.rb

```diff
@@ -10,19 +10,20 @@
   let(:session) { Session.create(uuid: user.uuid, token: 'abracadabra') }
 
   describe '#api_session_header' do
-    # much of this code comes from here: https://stackoverflow.com/a/51579296
+    # much of this code comes from here: https://gist.github.com/wildjcrt/6359713fa770d277927051fdeb30ebbf
     def decrypt_session_cookie(cookie)
-      salt = Rails.application.config.action_dispatch.authenticated_encrypted_cookie_salt
-      encrypted_cookie_cipher = 'aes-256-gcm'
+      config = Rails.application.config
+      salt   = config.action_dispatch.authenticated_encrypted_cookie_salt
```

Comment:
Gotta be honest I'd rather see one line here -- guessing this was for rubocop? I'd ignore it when it makes things HARDER to read

Reply:
config is reused in a couple places, but the salt only is used once so I just moved this line to where it's used